app/models/tag.rb

```diff
@@ -32,13 +32,13 @@ def to_param
     name
   end
 
-  def incrby(amount, status='live')
-    DateAggregation.new(total_charges_count_key).increment
-    DateAggregation.new(total_raised_amount_key).increment(amount)
+  def incrby(amount, status='live', charge_date: Time.zone.today)
```

Comment:
Can you update the `status` argument to use the same keyword approach we use for `charge_date`?